tests/cmd/main_test.py

```diff
@@ -62,6 +63,13 @@ def test_directory():
     assert _read_input_file() == method_to_function_output_contents == _read_output_file()
 
 
+def test_dry_run():
```

Comment:
you want the `capsys` pytest fixture: http://doc.pytest.org/en/latest/capture.html#accessing-captured-output-from-a-test-function